api/HTMLMediaElement.json

```diff
@@ -1757,7 +1760,10 @@
               "version_added": false
             },
             "firefox": {
-              "version_added": null
+              "version_added": "15"
```

Comment:
There we go, this `mozAudioCaptured` no longer looks like something that should be removed.